.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Acceptance tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - edited
+    - synchronize
```

Comment:
I would just leave `synchronize` and remove the rest - you don't need to rerun acceptance tests when someone edits a PR description, for example.